server/package.json

```diff
@@ -12,7 +12,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "bcrypt": "^4.0.0",
```

Comment:
Be careful when pushing new package.json updates, as they can cause some build issues. Although it seems fine here, just mentioning for future notice.

Reply:
Duly noted.